### PR TITLE
Add reduction parameter for RNNT loss

### PR DESCRIPTION
**Pull request by carolineechen**
add reduction argument for RNN transducer loss. options: mean (default), none, sum

### CODE REVIEW

File: torchaudio/prototype/rnnt_loss.py [hunk 1]
```diff
@@ -54,6 +56,11 @@ def rnnt_loss(
         fused_log_smax=fused_log_softmax,
         reuse_logits_for_grads=reuse_logits_for_grads,)
 
+    if reduction == 'mean':
+        return costs.mean()
+    elif reduction == 'sum':
+        return costs.sum()
```
Comment (mthrok) [inline torchaudio/prototype/rnnt_loss.py:62]: Can you perform argument check so that invalid value would raise an error?

**Review by vincentqb — APPROVED** [review]
LGTM, though let's wait on the tests to pass to merge.

**Comment by vincentqb** [comment]
offline discussion with @carolineechen -- the tests are passing locally, I'm ok with going ahead and merging this one.
